Core/include/Acts/Propagator/Propagator.hpp

```diff
@@ -30,6 +21,14 @@
 #include "Acts/Utilities/Definitions.hpp"
 #include "Acts/Utilities/Result.hpp"
 #include "Acts/Utilities/Units.hpp"
+#include "Acts/Utilities/detail/ReferenceWrapperAnyCompat.hpp"
```

Comment:
This would still need to go up top.

Reply:
Fixed for the `clang-format` 10 that I am using locally. This does not work with the `clang-format` version available on the CI. Should we bump it?

Reply:
We could. But you could probably also wrap this one in comments disabling the formatting.